__tests__/internal/move-after-handle-request/route-handlers/function-handler/basic-test.js

```diff
@@ -29,10 +29,13 @@ describe("Integration | Route handlers | Function handler", () => {
       throw "I goofed";
     });
 
+    let consoleSpy = jest.spyOn(console, "error").mockImplementation(() => {});
     let res = await fetch("/users");
     let data = await res.json();
 
-    expect(data.message).toBe("I goofed");
+    expect(consoleSpy).toHaveBeenCalled();
```

Comment:
What do you think of using `toHaveBeenCalledWith` to assert that it is being called with the right message?

Reply:
Good Idea! Changed it!